src/main/kotlin/net/lumalyte/lg/infrastructure/persistence/guilds/GuildRepositorySQLite.kt

```
@@ -641,4 +659,24 @@ class GuildRepositorySQLite(private val storage: Storage<Database>) : GuildRepos
+                allyHome?.worldId?.toString(),
+                allyHome?.position?.x,
+                allyHome?.position?.y,
+                allyHome?.position?.z,
+                guildId.toString()
+            )
+            // Update cache
+            guilds[guildId]?.let { guild ->
+                guilds[guildId] = guild.copy(allyHomeLocation = allyHome)
+            }
+            rowsAffected > 0
+        } catch (e: java.sql.SQLException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)